src/apollo/gql/gqlUser.ts

```diff
@@ -50,6 +50,7 @@ export const FETCH_USER_PROJECTS = gql`
 				totalDonations
 				totalReactions
 				verified
+				verificationStatus
```

Comment:
Is `verificationStatus` on `projectsByUserId` used anywhere?

Reply:
Not at the moment but it's still an important property of the schema. You want me to remove it?